packages/kobotoolbox/src/http.js

```diff
@@ -24,7 +24,7 @@ import * as util from './util';
  * Make a HTTP request to any KoboToolbox endpoint
  * @example <caption>Bulk updating of submissions</caption>
  * http.request("PATCH", `assets/${$.form_uid}/data/bulk/`, {
- *   body: {
+ *   data: {
```

Comment:
@hunterachieng this is a breaking change, it should be a major version bump.  Is there a good reason why we prefer `data` instead of `body`?

Reply:
@mtuchi  The code has not changed. It uses data, only the example and options docs are saying body. Let me update to a `major` change

Reply:
But if i have an existing workflow right now
`http.request('path', { body: {}})` that will no longer work because we no longer support `body`. Thats why i was saying this is a breaking change

Reply:
`body` has never worked. But I get your point. I have updated it